python/src/aiconfig/editor/server/server.py

```diff
@@ -811,9 +812,32 @@ def _op(
 
     signature: dict[str, Type[Any]] = {}
     return run_aiconfig_operation_with_request_json(
-        aiconfig, request_json, f"method_", _op, signature
+        aiconfig, request_json, f"method_{method_name}", _op, signature
```

Comment:
nice thanks!

Reply:
Actually upon further inspection, I think this should just be `method_name`, not `f"method_{method_name}"`: https://github.com/lastmile-ai/aiconfig/blob/f0f052231feee5f15843e2d88a70f7b406527dd5/python/src/aiconfig/editor/server/server_utils.py#L501-L503